workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+
+Once you have added widgets to your dashboard, you will be taken directly to your dashboard:
+
+<Note> Data will appear momentarily after you have created your new dashboard. </Note>
+
+![](https://lh7-us.googleusercontent.com/rg4LfJetovyC5UMrgE7H3aN3UUuK0K2kbbHp5LYgGHLs12M-48jW3OtN7RlPA0sJnTznOpcdtNFqghehlroWY7lhsw1eQsE1ZSKXSqf8e-EUNX99RzcCr24ScSxFiXhp27NvLokBGwV9PMuKWk8jwTY)
+
+# Create New Widget
+
+Navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
+
+![](https://lh7-us.googleusercontent.com/Qn2pnY24-2CuG7moVbGCi4UqO0aoIMdqNjsYaSipLHo3gfUOhD_z0vchbVyMNrN3zb6gPZeCAkkoJWmndzgqtcZVVFz9jzJYD_gkhP3qWNytupLefOGdMGLhd5Z9SjTvxqXbMmM1x6XYF0QzXK9LNn8)
+
+![](https://lh7-us.googleusercontent.com/Oe2a3IYXnQ4JTYXRbzMe9c7RZSHXFySFLJGFZCxWhJIFIvlNM2hOmGgQdCapl2h8Hk5-FhEWRwALQxZ1lJ2BGJtnvQnXA_r0TCqX5o3OeXwJnAqQrR1wkm77-uZLmL5V7QTiGpPr4Uhc6SRvjaO3Yjg)
+
+### Chart Types
+
+Choose from one of the following types of charts for your widget:
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+| __Chart Type__     | __Description__ |
+| :-------------     | :------------|
+| __Timeseries__     | Displays data collected over time |
+| __Bar__            | Displays data proportional to available data |
+| __List__           | Displays data in a table, updating metrics in real time |
+| __Query Value__    | Displays data as a single value, updating metrics in real time |
+| __Pie Chart__      | Displays data in the form of a pie chart|
+| __Scatter Plot__   | Displays relationship/correlation between two metrics |
+| __Tree Plot__      | Displays hierarchal relationship of your data as a function of size and color |
+| __Top List Chart__ | Displays the highest-ranking elements or categories based on predefined metric criteria|
+| __Heatmap Chart__  | ---------- |
+| __Hexagon Chart__  | Displays host and container metrics as a function of color  |
+
+### Create Your Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within your query.
+
+Think of your queries like 2-dimensional arrays, where __Aggregate By__ is a function that performs a transformation on each subarray and __Group By__ performs an aggregate function on the aggregate outputs across all arrays. 
+
+![](https://lh7-us.googleusercontent.com/8sltD2VI-oNKeR0PPF2cp1rx5ktjK-KfXnF-vOUzvfXbcDElFeTKEXmsx6wm3PDN95xl1IjbcOwyX9Oq5CXoKVcjroG1V0eq3JeaL0XP8cvUU4yqIKKyFQa2t7dDrpSJJKaM6emthaYHDvGNSr3ztOs)
```

Comment:
Can we work with Uttam on a pretty picture here, w/ transparent background that works in light and dark mode?

Reply:
FWIW we can push this in the first version but we should ask uttam for a prettier pic to push soon after we go live